girder_worker/app.py

```diff
@@ -231,15 +256,29 @@ def girder_before_task_publish(sender=None, body=None, exchange=None,
                 headers['jobInfoSpec'] = utils.jobInfoSpec(job)
 
             except ImportError:
-                # TODO: Check for self.job_manager to see if we have
-                #       tokens etc to contact girder and create a job model
-                #       we may be in a chain or a chord or some-such
-                pass
+                gc = GirderClient(apiUrl=headers['girder_api_url'])
+                gc.token = headers['girder_client_token']
```

Comment:
error if girder_client_token is not in headers. shouldn't this be coming from task.request anyways?